experimenter/experimenter/nimbus-ui/src/lib/experiment.ts

```diff
@@ -50,16 +53,19 @@ export function getStatus(
       statusNext === NimbusExperimentStatusEnum.COMPLETE,
     updateRequested:
       isRollout === true &&
+      isRolloutDirty === true &&
       status === NimbusExperimentStatusEnum.LIVE &&
       publishStatus === NimbusExperimentPublishStatusEnum.REVIEW &&
       statusNext === NimbusExperimentStatusEnum.LIVE,
     updateRequestedApproved:
       isRollout === true &&
+      isRolloutDirty === true &&
```

Comment:
once request get approved, this should go back to false??

Reply:
this will go back to false once the update is fully approved in experimenter _and_ remote settings (which means it goes through `publish_status`=DIRTY -> REVIEW -> APPROVE -> WAITING -> IDLE). This means that the kinto tasks will set `publish_status` back to IDLE and `is_rollout_dirty` to False:

Reply:
okay okay makes sense 💟 , thank you @eliserichards